resources/js/pages/home/Components/Story/HomeStory.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import StoryContent from './StoryContent';
+import './story.css';
+
+
+export const HomeStory = () => {
+    return (
+        <>
+            <div className="my-3">
+                <div className=" overflow-x-auto scrollbar-hide">
+                    <div className='flex w-max'>
+                        <div className="mx-1 p-1 rounded-full bg-gradient-to-r from-yellow-400 via-red-500 to-purple-500">
```

Comment:
you already have a story model, use it to build these stories via a map don't copy past and use fake data in components